models/chain_info/chain_info_dune_chains.sql

```diff
@@ -0,0 +1,23 @@
+{{
+  config(
+    schema = 'chain_info'
+    , alias = 'dune_chains'
+    , tags=['static']
+  )
+}}
+
+
+SELECT dune_name
+     , chain_id
+FROM (
+  VALUES
+    ('ethereum', 1),
+    ('optimism', 10),
+    ('bnb', 56),
+    ('gnosis', 100),
+    ('polygon', 137),
+    ('fantom', 250),
+    ('arbitrum', 42161),
+    ('avalanche_c', 43114),
+    ('solana', 1399811149)
```

Comment:
```suggestion
```

Reply:
We can add back solana id when we have more confident source.